snoop/configuration.py

```diff
@@ -62,8 +64,9 @@ def install(
         - `full_file`: The full path to the file (also shown anyway when the function is called).
         - `function`: The name of the current function.
         - `function_qualname`: The qualified name of the current function.
-        
-        If you want a custom column, please open an issue to tell me what you're interested in! In the meantime, you can pass a list, where the elements are either strings or callables. The callables should take one argument, which will be an `Event` object. It has attributes `frame`, `event`, and `arg`, as specified in [`sys.settrace()`](https://docs.python.org/3/library/sys.html#sys.settrace), and other attributes which may change. 
+
+        If you want a custom column, please open an issue to tell me what you're interested in! In the meantime, you can pass a list, where the elements are either strings or callables. The callables should take one argument, which will be an `Event` object. It has attributes `frame`, `event`, and `arg`, as specified in [`sys.settrace()`](https://docs.python.org/3/library/sys.html#sys.settrace), and other attributes which may change.
+    - `pformat`: set the pretty formatting function `pp` uses. Default is to use the first of `prettyprinter.pformat`, `pprintpp.pformat` and `pprint.pformat` that can be imported.
```

Comment:
Please also mention the pformat argument here and under Output configuration.

Reply:
It is mentioned there (?) I pushed a new commit which adds it to the README's Output configuration section.

Reply:
lol my brain only saw mention of the libraries and not the argument.